sentry-android-core/src/main/java/io/sentry/android/core/AndroidContinuousProfiler.java

```diff
@@ -103,7 +104,10 @@ private void init() {
             logger);
   }
 
-  public synchronized void start(final @NotNull TracesSampler tracesSampler) {
+  @Override
+  public synchronized void startProfileSession(
```

Comment:
Would it make sense to return a boolean, indicating if profiling actually started or not?

Reply:
maybe yes, but at the moment it wouldn't be used anywhere. We can always add it later imho